README.md

```diff
@@ -155,6 +155,22 @@ For instance, you can ignore all the npm modules:
 ignoreFilenames: ['node_modules'],
 ```
 
+### `additionalLibraries`
+
+This option gives the possibility to remove other `propTypes` in addition to the canonical `prop-types`.
+
+For instance, by default
+```js
+import PropTypes from 'prop-types'
+import ImmutablePropTypes from 'react-immutable-proptypes'
+```
+will result in `propTypes` not to be removed; with:
```

Comment:
I would expect all the `propTypes` to be removed no matter the import done. Did you notice a different behavior?